paper/abstract.md

```diff
@@ -0,0 +1,20 @@
+
+
+As higher education decision makers increasingly turn to statistical
+and machine learning algorithms to predict student success, they often
+do so without a clear understanding of how these algorithms work or
+the tradeoffs made when basing decisions upon their predictions. Of
+particular concern are hidden biases inherent in an uncritical use of
+these algorithms that negatively affect already marginalized student
+populations. In this project we use `R` to transparently build a
+number of predictive algorithms and show how decisions based on their
+results might affect already underrepresented student groups in higher
```

Comment:
Would anyone be opposed to changing underrepresented to underserved here?

Reply:
I think all else looks good!

Reply:
@ksalazar3, are you thinking to change both instances of "underrepresented" to "underserved" or just this one you highlight? I'm fine either way.

Reply:
Both. I think underrepresented just puts onus on students rather than institutions/decision makers

Reply:
Yeah, then let's change both. I'll update.